src/Collections-Unordered/Set.class.st

```diff
@@ -144,6 +144,12 @@ Set >> collect: aBlock [
 	^ newSet
 ]
 
+{ #category : 'copying' }
+Set >> copyEmpty [
+
+	^ self species new: self size
```

Comment:
Setting the size of the copy to the size of the original goes against the way `copyEmpty` is implemented in all other classes, where it is often explicitly (otherwise implicitly) set to 0.

Reply:
Indeed, thanks for the comments